tests/testHSCColorterms.py

```diff
@@ -0,0 +1,97 @@
+import os
+import numbers
+import unittest
+
+import lsst.utils.tests as utilsTests
+import lsst.pipe.tasks.photoCal as photoCal
+
+class ColortermOverrideTestCase(unittest.TestCase):
+    def setUp(self):
+        """Test that colorterms specific to HSC override correctly"""
+        colortermsFile = os.path.join(os.environ["OBS_SUBARU_DIR"],"config","hsc","colorterms.py")
+        self.photoCalConf = photoCal.PhotoCalConfig()
+        self.photoCalConf.colorterms.load(colortermsFile)
+
+    def testHSCColorterms(self):
+        """Test that the colorterm libraries are formatted correctly"""
+        refBands = ["g","r","i","z","y"]
+        hscBands = ["g","r","i","z","y"]
```

Comment:
Why both? If you need both, make it explicit with a copy `hscBands = refbands[:]`.